opentreemap/treemap/models.py

```diff
@@ -1071,6 +1071,12 @@ def delete_with_user(self, *args, **kwargs):
         thumb.delete(False)
         image.delete(False)
 
+    @classmethod
+    def display_name(cls, instance):
```

Comment:
Both Tree and Plot use the `_terminology` API despite also being not configurable. I don't see a reason to introduce this layer on top of terminology, but if we do, we should convert Tree and Plot as well.